static/app/views/organizationGroupDetails/groupEvents.spec.jsx

```diff
@@ -181,6 +181,32 @@ describe('groupEvents', function () {
       {context: routerContext, organization}
     );
     const perfEventsColumn = screen.getByText('transaction');
+    expect(discoverRequest).toHaveBeenCalledWith(
+      '/organizations/org-slug/events/',
+      expect.objectContaining({
+        query: expect.objectContaining({query: 'performance.issue_ids:1 '}),
+      })
+    );
     expect(perfEventsColumn).toBeInTheDocument();
   });
+
+  it('renders new events table if error', function () {
```

Comment:
Add one more test on the other side of the feature flag and assert that the non-flagged behaviour still happens, we can remove the tests along with the flag once it's fully rolled out.

Reply:
done!